omod/src/main/webapp/resources/htmlforms/mexico/consult.xml

```diff
@@ -949,14 +961,18 @@
         </h1>
     </ifMode>
 
-    <section id="analysis" sectionTag="fieldset" headerTag="legend" headerStyle="title">
-        <div class="section-container">
-            <div id="data-collection">
-                <p>
-                    <label><uimessage code="emr.consult.freeTextComments"/></label>
-                    <obs conceptId="PIH:CLINICAL IMPRESSION COMMENTS" style="textarea" rows="5"/>
-                </p>
-                <encounterDiagnoses selectedDiagnosesTarget="#encounter-diagnoses-target"/>
```

Comment:
There were two problems: I don't know what either `encounterDiagnoses` or `encounterDiagnosesByObs` is, but the former doesn't seem to add diagnoses to the `encounter-diagnosis-target`. The `encounter-diagnosis-target` also didn't exist.

This is all correct in `sections-dx`. There are no other usages of `encounterDiagnoses` in PIH EMR.